<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectroniqueForm</class>
 <widget class="QDialog" name="ElectroniqueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CatalogueForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Garantie</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconditionne</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDesc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Smartphone Galaxy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrix">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>499.90</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PRD-SMAR-60</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGaran">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRecon">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>ElectroniqueForm</receiver>
   <slot>slotOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOK()</slot>
 </slots>
</ui>
